libblkid/src/superblocks/bcache.c

```diff
@@ -316,7 +323,8 @@ static int probe_bcachefs(blkid_probe pr, const struct blkid_idmag *mag)
 		return BLKID_PROBE_NONE;
 
 	sb_size = BCACHEFS_SB_FIELDS_OFF + BYTES(bcs);
-	if (sb_size > BCACHEFS_SB_MAX_SIZE)
+	sb_max_size = BCACHEFS_SECTOR_SIZE * 2 << (bcs->layout.sb_max_size_bits - 1);
```

Comment:
Why `* 2` and `-1`? Aren't those cancelling each other out?

Reply:
You're right, fixed.